build.gradle

```diff
@@ -1,5 +1,5 @@
 ext {
-    dashjVersion = "0.16.2"
+    dashjVersion = "0.17-SNAPSHOT"
```

Comment:
Why not just `0.17`?

Reply:
0.17 is not completed.  It will be completed when the redesign is ready to be released.  Additionally, SNAPSHOT allows us to host these snapshot builds on a maven type repo that hosts snapshots.  This will allow the rest of the developers to not worry about building dashj locally on their machine in order to build the app when 0.17 is required.